src/odemis/cli/main.py

```
@@ -715,35 +717,46 @@ def move_abs(comp_name, moves, check_distance=True, to_radians=False):
+                raise ValueError("Actuator %s has no axis '%s'" % (comp_name, axis_name))
     except (TypeError, AttributeError):
         raise ValueError("Component %s is not an actuator" % comp_name)
 
     try:
-        if axis_name not in component.referenced.value:
-            raise AttributeError()  # immediately caught
+        for axis_name in axis_names:
+            if axis_name not in component.referenced.value:
+                raise AttributeError()  # immediately caught
     except (TypeError, AttributeError):
         raise ValueError("Axis %s of actuator %s cannot be referenced" % (axis_name, comp_name))
```

Ruff (0.15.21)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)